css/components/badge.css

```diff
@@ -0,0 +1,94 @@
+@import "../helpers/spacing.css";
+@import "../utilities/variables.css";
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+
+.badge-wrapper{
+    display: inline-block;
+}
+
+.status-online{
+    background-color: var(--green);
+}
+
+.status-offline{
+    background-color: var(--grey);
+}
+
+.status-away{
+    background-color: var(--yellow); 
+}
+
+.status-dnd{
+    background-color: var(--red);
+}
+
+.badge-status{
+    padding: var(--size-4);
+    top:calc(2.7*var(--size-7));
+    left:calc(3*var(--size-7));
+}
+
+.badge-status-sm{
+    padding: var(--size-5);
+    top:calc(4*var(--size-7));
+    left:calc(4.5*var(--size-7));
+}
+
+.badge-status-md{
+    padding: var(--size-6);
+    top:calc(5.5*var(--size-7));
+    left:calc(6*var(--size-7));
+}
+
+.badge-status-lg{
+    padding: var(--size-7);
+    top:calc(7*var(--size-7));
+    left:calc(7.5*var(--size-7));
+}
+
+.badge-status-xl{
+    padding: var(--size-8);
+    top:calc(8.5*var(--size-7));
+    left:calc(9*var(--size-7));
+}
+
+.badge-status-xxl{
+    padding: var(--size-9);
+    top:calc(10*var(--size-7));
+    left:calc(10.5*var(--size-7));
+}
+
+.count-green{
+    background-color: var(--green);
+}
+
+.count-red{
+    background-color: var(--red);
+}
+
+.count-grey{
+    background-color: var(--grey);
+}
+
+.count-blue{
+    background-color: var(--blue);
+}
+
+.badge-count{
+    color: var(--white);
+    width:calc(var(--size-10));
+    height:calc(var(--size-10));
+    top:-50%;
+    left:50%; 
+    font-weight: var(--weight-bold);
+}
+
+.badge-count-overflow{
+    top:-40%;
+    left:50%; 
+}
+
+.badge-count-color-pos{
+    top:-20%;
+    left:50%; 
+}
```

Comment:
Add new line here

Reply:
I have added new line there.

Reply:
@Hamdrive mentioned link

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log